app/main/db/queries.py

```diff
@@ -637,58 +556,51 @@ def _get_file_metadata_query(file_id: uuid.UUID):
 
-    query = db.session.query(
-        sub_query.c.file_id,
-        sub_query.c.file_name,
-        sub_query.c.file_path,
-        sub_query.c.citeable_reference,
-        sub_query.c.alternative_title,
-        sub_query.c.description,
-        sub_query.c.alternative_description,
-        sub_query.c.closure_type,
-        func.to_char(
-            sub_query.c.closure_start_date,
-            current_app.config["DEFAULT_DATE_FORMAT"],
-        ).label("closure_start_date"),
-        sub_query.c.closure_period,
-        func.to_char(
-            sub_query.c.opening_date,
-            current_app.config["DEFAULT_DATE_FORMAT"],
-        ).label("opening_date"),
-        func.to_char(
-            sub_query.c.date_last_modified,
-            current_app.config["DEFAULT_DATE_FORMAT"],
-        ).label("date_last_modified"),
-        sub_query.c.foi_exemption_code,
-        sub_query.c.transferring_body_id,
-        sub_query.c.transferring_body,
-        sub_query.c.series_id,
-        sub_query.c.series,
-        sub_query.c.consignment_id,
-        sub_query.c.consignment_reference,
-        sub_query.c.file_reference,
-        sub_query.c.former_reference,
-        sub_query.c.translated_title,
-        sub_query.c.held_by,
-        sub_query.c.legal_status,
-        sub_query.c.rights_copyright,
-        sub_query.c.language,
-    )
+    query = (
+        db.session.query(
+            sub_query.c.file_id,
+            sub_query.c.file_name,
+            sub_query.c.file_path,
+            sub_query.c.citeable_reference,
+            sub_query.c.alternative_title,
+            sub_query.c.description,
+            sub_query.c.alternative_description,
+            sub_query.c.closure_type,
+            func.to_char(
+                sub_query.c.closure_start_date,
+                current_app.config["DEFAULT_DATE_FORMAT"],
+            ).label("closure_start_date"),
+            sub_query.c.closure_period,
+            func.to_char(
+                sub_query.c.opening_date,
+                current_app.config["DEFAULT_DATE_FORMAT"],
+            ).label("opening_date"),
+            func.to_char(
+                sub_query.c.date_last_modified,
+                current_app.config["DEFAULT_DATE_FORMAT"],
+            ).label("date_last_modified"),
+            sub_query.c.foi_exemption_code,
+            sub_query.c.file_reference,
+            sub_query.c.former_reference,
+            sub_query.c.translated_title,
+            sub_query.c.held_by,
+            sub_query.c.legal_status,
+            sub_query.c.rights_copyright,
+            sub_query.c.language,
+            Body.Name.label("transferring_body"),
+            Series.Name.label("series"),
+            Consignment.ConsignmentReference.label("consignment_reference"),
+        )
+        .join(File.consignment)
+        .join(Consignment.series)
+        .join(Series.body)
+    ).where(
+        sub_query.c.file_id == File.FileId
+    )  # using custom join with subquery
```

Comment:
I think this was just a leftover comment, since this isnt a custom join, its a filter.

So can remove?